libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Dialogs/Microsoft.ConfirmInput.schema

```diff
@@ -82,7 +82,7 @@
                     }
                 },
                 {
-                    "$ref": "schema:#/definitions/equalsExpression"
+                    "$ref": "schema:#/definitions/stringExpression"
```

Comment:
Does this affect backwards compat with existing declarative bots?

Reply:
Hi @EricDahlvang, we tested this with a composer bot, setting the choices and the choice options as an object and as an expression.
We compared the behavior with version 4.16.0 and the bot behaves the same, except that now, expressions are supported for choice options.
![image](https://user-images.githubusercontent.com/44245136/174135712-543718e2-7d91-4bc7-a9dc-19852c9ab21d.png)
![image](https://user-images.githubusercontent.com/44245136/174135723-efbb328f-0603-4bf0-9206-4cf27bc9b088.png)